src/environments/environment.staging.ts

```diff
@@ -0,0 +1,5 @@
+export const environment = {
+  production: true,
+  hostName: 'http://ec2-3-15-173-123.us-east-2.compute.amazonaws.com/api',
+  googleMapsApiKey: 'AIzaSyDSPR8JtXAR1_vnYJHpVZPSLYr5Y6_fgDs'
```

Comment:
Shouldn't this be encrypted in some way? For eg using `git-crypt`?

Reply:
Following the same approach as production: https://github.com/chennaitricolor/wastexchange-fe/blob/master/src/environments/environment.prod.ts

I think we are restricting the domain on google cloud console. @subbu3012 can confirm?

Reply:
I have changed staging to use dev google maps API key.